packages/sitecore-jss-nextjs/src/services/graphql-sitemap-service.ts

```diff
@@ -208,8 +215,18 @@ export class GraphQLSitemapService {
         debug.sitemap('fetching sitemap data for %s', language);
 
         return this.fetchLanguageSitePaths(this.query, args).then((results) => {
-          return results.map((item) =>
-            formatStaticPath(item.path.replace(/^\/|\/$/g, '').split('/'), language)
+          const aggregatedPaths: string[] = [];
+          results.forEach((item) => {
+            aggregatedPaths.push(item.path);
+            if (item.personalize?.variantIds.length) {
+              aggregatedPaths.push(
+                ...item.personalize?.variantIds.map((varId) => `/${varId}${item.path}`)
```

Comment:
Even though I will eventually switch this to use the helper function as we discussed, the format should be
```suggestion
                ...item.personalize?.variantIds.map((varId) => `/_segmentId_${varId}${item.path}`)
```

Reply:
Added - segmentId brought out into a const for easier refactoring (when needed) and better readability